src/S1_fft_cc_MPI.py

```diff
@@ -369,7 +382,8 @@
 
             #---------------keep daily cross-correlation into a hdf5 file--------------
             if input_fmt == 'h5':
-                tname = tdir[ick].split('/')[-1]
+                tname = tdir[ick].split('\\')[-1]
+
```

Comment:
To avoid platform specific paths:
```suggestion
                tname = os.path.split(tdir[ick])[1]

```
But please test this first :)